config/environments/development.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
   
+  # docker specific development setup
+  if ENV.fetch('DOCKERIZED', false) == 'true'
+    # setup hot reload
+    config.middleware.insert_after(ActionDispatch::Static, Rack::LiveReload)
+    # whitelist ip to enable web console
+    host_ip = `/sbin/ip route|awk '/default/ { print $3 }'`.strip
```

Comment:
Very nice. Its not every day one sees `awk` used. And especially not from within ruby! Very elegant solution.

In the past I have seen this setup web console simply plugged in as a entry where you have deleted it from.
This is a far better solution as its dynamic. Super nice @dtt101

Reply:
Totally google searched it, but also thought it was a nice solution! So good to get the review :)